.github/workflows/publish.yml

```diff
@@ -1,7 +1,7 @@
 ---
 
 name: Publish delb
-on:
+on:  # yamllint disable-line
```

Comment:
what is this needed for?

Reply:
it's not needed as yamllint's default just produces warning in this case. to get rid of them the `on` keys would have to be enclosed in quotes.